pyupgrade.py

```diff
@@ -1087,6 +1090,16 @@ class FindPy3Plus(ast.NodeVisitor):
         'WindowsError',
     ))
 
+    OS_ERROR_ALIAS_MODULES = frozenset((
+        'mmap',
+        'select',
+        'socket',
+    ))
+
+    FROM_IMPORTED_MODULES = frozenset(
+        itertools.chain(('six',), OS_ERROR_ALIAS_MODULES),
```

Comment:
no need for `chain`, just use `{'six'} | OS_ERROR_ALIAS_MODULES`

Reply:
or `OS_ERROR_ALIAS_MODULES.union(('six',))`

Reply:
another idea is to ditch these constants entirely and record all the `from` imports

Reply:
will do the union approach for now just to be slightly more DRY